js/app.js

```diff
@@ -396,51 +396,62 @@ app.controller('PitController', function ($scope, $http, $routeParams, $location
 		}
 	}
 
-	$http.get('php/getPitData.php', {
-		params: {
-			teamNumber: $routeParams.teamNumber
-		}
-	}).then(function (response) {
-		$scope.data = response.data;
-
-		if ($scope.data.teamInfo.name != null) {
-			$scope.name = $scope.data.teamInfo.name + ($scope.data.teamInfo.name[$scope.data.teamInfo.name.length - 1] == "s" ? "'" : "'s");
-		} else {
-			$scope.name = $scope.teamNumber + "'s";
+	$http.get('php/getScouterTeams.php',).then(function (response) {
+		$scope.teams = response.data;
+		if ($scope.teams.length) {
+			$scope.currentTeam = $scope.teams[0];
+			$scope.loadData($scope.currentTeam.team_number);
 		}
+	});
 
-		if (response.data.commentSection != null) {
-			for (var i = 0; i < response.data.commentSection.length; i++) {
-				$scope.pitData.comments.push({
-					comment: response.data.commentSection[i]['Pit Scouters Comments'],
-					commenter: response.data.commentSection[i]['Pit Scouter'],
-					timeStamp: response.data.commentSection[i]['timestamp']
-				});
+	$scope.loadData = function(scoutingTeam) {
+		$http.get('php/getPitData.php', {
+			params: {
+				teamNumber: $routeParams.teamNumber,
+				scoutingTeam: scoutingTeam
 			}
-		} else {
-			$scope.noComments = true;
-		}
-		if (response.data.pics != null) {
-			for (var i = 0; i < response.data.pics.length; i++) {
-				$scope.pitData.pictures.push({
-					pictureNumber: response.data.pics[i]['Picture Number'],
-					photographer: response.data.pics[i]['Pit Scouter'],
-					timeStamp: response.data.pics[i]['timestamp']
-				});
+		}).then(function (response) {
+			$scope.data = response.data;
+
+			if ($scope.data.teamInfo.name != null) {
+				$scope.name = $scope.data.teamInfo.name + ($scope.data.teamInfo.name[$scope.data.teamInfo.name.length - 1] == "s" ? "'" : "'s");
+			} else {
+				$scope.name = $scope.teamNumber + "'s";
 			}
-			$scope.picIndex = 0;
-		} else {
-			$scope.noPictures = true;
-		}
 
-		$(document).ready(function () {
-			if ($scope.noComments && $scope.noPictures) {
-				$("#errorModal").modal("show");
+			if (response.data.commentSection != null) {
+				for (var i = 0; i < response.data.commentSection.length; i++) {
+					$scope.pitData.comments.push({
```

Comment:
Before you start adding comments clear out the comments array, you can call the splice function and pass 0 to it to remove all entries, e.g. $scope.pitData.comments.splice(0). Somewhere before that for loop